fastlane/jetpack_metadata/en-US/name.txt

```diff
@@ -0,0 +1 @@
+Jetpack
```

Comment:
The other locales don't have this file, but it's not "wrong" to have it, so I left it here.

`name.txt` was my first (failed) attempt to get `deliver` to upload the release notes.

Reply:
I'm pretty confident that having it wouldn't have disrupted the upload flow for the other languages, but since I likely won't be the one running the next upload, I want to keep the changes to a minimum.